tests/views_wpunit/Tribe/Events/Views/V2/Day_ViewTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Tribe\Events\Views\V2;
+
+use Tribe\Events\Views\V2\Views\Day_View;
+use Tribe\Test\PHPUnit\Traits\With_Post_Remapping;
+use Tribe__Events__Main as TEC;
+use DateTime;
+
+/**
+ * Class Month_ViewTest
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2
+ */
+class Day_ViewTest extends \Codeception\TestCase\WPTestCase {
```

Comment:
The same test should exist in the `ct1_integration` suite to make sure the cold and warm cache values are correct.